students/Wieslaw_Pucilowski/lesson03/mailroom.py

```diff
@@ -85,18 +72,29 @@ def send_thank_you():
 		indx=donors.index(name)
 		donors_hist[indx].append(float(donation))
 		greetings(donors_hist[indx])
-	return # main_menu()
-	
+	#return 
+
+def bubblesort(ListOfLists):
+        for i in range(0,len(ListOfLists)):
+            madeSwap = False
+            for j in range(0,len(ListOfLists)-1):
+                if sum(ListOfLists[j][1:]) > sum(ListOfLists[j+1][1:]):
+                    temp = ListOfLists[j+1]
+                    ListOfLists[j+1] = ListOfLists[j]
+                    ListOfLists[j] = temp
+                    madeSwap = True
+            if not madeSwap:
+                return ListOfLists
+        return ListOfLists
```

Comment:
@wpucilow couple comments 🙂 
* you have mixed spacing here.. make sure to _only_ use 4 space indent
* per pep8 style guidelines, var names should follow snake_case convention
* you should use `sorted` method to sort your donors, no need to implement your own sorting logic; please check out this doc (sorted + itemgetter combo): https://wiki.python.org/moin/HowTo/Sorting